python/cudf/cudf_pandas_tests/test_cudf_pandas.py

```diff
@@ -1243,6 +1243,34 @@ def my_apply(df, unused):
     tm.assert_series_equal(result, expected)
 
 
+@pytest.mark.parametrize(
+    "op",
+    [
+        "__iadd__",
+        "__iand__",
+        "__ifloordiv__",
+        "__imod__",
+        "__imul__",
+        "__ior__",
+        "__ipow__",
+        "__isub__",
+        "__itruediv__",
+        "__ixor__",
+    ],
+)
+def test_inplace_ops(op):
+    xdf1 = xpd.DataFrame({"a": [10, 11, 12]})
+    xdf2 = xpd.DataFrame({"a": [1, 2, 3]})
+
+    df1 = pd.DataFrame({"a": [10, 11, 12]})
+    df2 = pd.DataFrame({"a": [1, 2, 3]})
+
+    xdf1 = getattr(xdf1, op)(xdf2)
+    df1 = getattr(df1, op)(df2)
+
+    tm.assert_equal(xdf1, df1)
```

Comment:
You don't _have_ to mutate the existing object when calling in-place operators explicitly. [As the docs say:](https://docs.python.org/3/library/operator.html#in-place-operators)

> Another way to put it is to say that `z = operator.iadd(x, y)` is equivalent to the compound statement `z = x; z += y`.

For that reason, I would use our normal `expected`/`actual` approach to naming:
```suggestion
    actual = getattr(xdf1, op)(xdf2)
    expected = getattr(df1, op)(df2)

    tm.assert_equal(actual, expected)
```